js/main.js

```diff
@@ -18,4 +20,47 @@ const getRandomNumberComma = (min, max, comma) => {
 
 export {getRandomNumber, getRandomNumberComma}
 
+const randomValues = (arr) => {
+  return arr.slice(getRandomNumber(1, arr.length - 1));
+}
+
+const TYPE = ['palace', 'flat', 'house', 'bungalow'];
+
+const PHOTOS = ['http://o0.github.io/assets/images/tokyo/hotel1.jpg', 'http://o0.github.io/assets/images/tokyo/hotel2.jpg', 'http://o0.github.io/assets/images/tokyo/hotel3.jpg.'];
+
+const FEATURES = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+
+const CHEKOUT = ['12:00', '13:00', '14:00'];
+
+const CHECKIN = ['12:00','13:00','14:00'];
+
+const TITLE = ['Маленькая студия','Большая студия','Аппартаменты','Комната','Бунгало','Уютный домик','Домик у побережья'];
+
+const DESCRIPTION= ['Цена - качество','Красивый вид','Рядом магазины','Близко к остановке'];
+
+const MakeRandomItems = () => {
```

Comment:
```suggestion
const makeRandomItems = () => {
```
Как минимум нужно убрать большую букву вначале. Функции принято писать с маленькой

Так же написано `items` хоть в данном случае возвращается только один айтем

Название лучше сделать какое-то более говорящее какой именно айтем. Вроде `createAd` для одного объявления и `createAd` для многих